### ISSUE TITLE
Search behavior on obo files not as expected

### ISSUE THREAD

**Issue by deepakunni3** [INCATools/ontology-access-kit#88]
I noticed this first when going through the tutorials and running search via the cli:

```sh
runoak --input obolibrary:fbbt.obo search 'wing vein'
```

What is expected in the tutorial is:
```sh
FBbt:00004751 ! wing vein
FBbt:00004754 ! axillary vein
FBbt:00004759 ! wing vein L1
FBbt:00004760 ! wing vein L2
```

But what is returned by OAK is:

```sh
FBbt:00004751 ! wing vein
```

This behavior is seen on:
- `main` (ref `9bd64d9`)
- v0.1.19

After a bit of tinkering I realized that this might be because the default search behavior might have changed from `SearchTermSyntax.REGULAR_EXPRESSION` to `SearchTermSyntax.STARTS_WITH`.

**Comment by cmungall** [comment]
You are correct, the most recent release unified most search behavior. Exception is the blaze graph and bioportal endpoints that default to relevance ranked
